students/kulon_aleksander/lesson_02/knight_move.py

```diff
@@ -0,0 +1,12 @@
+cell1_x = int(input())
+cell1_y = int(input())
+
+cell2_x = int(input())
+cell2_y = int(input())
+
+if ((cell1_y + 2 == cell2_y) or (cell1_y - 2 == cell2_y)) and ((cell1_x + 1 == cell2_x) or (cell1_x - 1 == cell2_x)):
```

Comment:
A bit complicated. This and next "if" could be simplified, so instead two "if" statements you can have something like this:
if abs(cell1_x - cell2_x) == 2 and abs(cell1_y - cell2_y) == 1 or \
       abs(cell1_y - cell2_y) == 2 and abs(cell1_x - cell2_x) == 1:

In general results you got are ok so no need to fix it.